examples/server.mjs

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+import { load } from "@azure/app-configuration-provider";
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+const appConfig = await load(connectionString, {
+    featureFlagOptions: {
+        enabled: true,
+        selectors: [{
+            keyFilter: "*"
+        }],
+        refresh: {
+            enabled: true,
+            refreshIntervalInMs: 10_000
+        }
+    }
+});
+
+import { ConfigurationMapFeatureFlagProvider, FeatureManager } from "@microsoft/feature-management";
+const featureProvider = new ConfigurationMapFeatureFlagProvider(appConfig);
+const featureManager = new FeatureManager(featureProvider);
+
+app.get('/', (req, res) => {
+    res.send('Welcome to the original feature!');
+});
+
+app.get('/beta', async (req, res) => {
+    appConfig.refresh();
```

Comment:
Will refresh fail and return error? If so, can we add error handling?

Reply:
refresh will return a `Promise`. If any error ocurrs during the refresh, the returned `Promise` will be rejected. It will not break the thread. It is fine to not have error handling here. I want to keep the example simple. The [example]( https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/refresh.mjs) in JS provider repo and our public [doc](https://learn.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-javascript?tabs=configuration-map) don't have error handling either.

Reply:
Agree not to add error handling, just to clearly demonstrate what is minimal to get refresh working. 

Or at most `appConfig.refresh().catch(console.warn);`, also indicating this is a async promise.